contracts/src/v0.8/dev/automation/2_1/KeeperRegistry2_1.sol

```diff
@@ -167,7 +166,7 @@ contract KeeperRegistry2_1 is KeeperRegistryBase2_1, OCR2Abstract, Chainable, ER
             reimbursement + premium,
             upkeepTransmitInfo[i].gasUsed,
             upkeepTransmitInfo[i].gasOverhead,
-            upkeepTransmitInfo[i].upkeepTriggerID
+            report.triggers[i]
```

Comment:
I am not sure if this will fully unblock @amirylm for log recoverer. We'll need to lookup logs by (logTxHash, LogIndex) and as such we'll need to do a full scan of these logs in DB to understand if a particular logTxHash, LogIndex was done?

Reply:
perfect wanna just put `logTxHash, LogIndex` in the event. because this will also make o11y easier.

Reply:
+ also make it human readable. But not sure if it should be part of upkeepPerformed log itself (as that is generic for triggers)